lib/serviceCatalogue.js

```diff
@@ -1,12 +1,22 @@
 import co from 'co';
 import https from 'https';
+import {EOL as newline} from 'os';
+
+const errMsg = (apiDomain) => `
+Could not connect to ${apiDomain}.
+
+Please make sure you're config details are correct
+and you are connected to a VPN if you need to be.`;
 
 /**
  * Get projects from multiple endpoints
  * @param paths
  * @returns {Array}
  */
 const getProjects = apiConfig => {
+  process.stdout.write(newline);
```

Comment:
It can and it was, but it then has to be accounted for in the test and it felt unnecessary. That's why I moved it out.